packages/edge/js/Edge.js

```diff
@@ -39,6 +39,32 @@ module.exports = {
    * if no handles were returned for the given experienceEvent
    */
   sendEvent(experienceEvent: ExperienceEvent): Promise<Array<EdgeEventHandle>> {
-     return Promise.resolve(RCTAEPEdge.sendEvent(experienceEvent));
-  }
+   const sentEventPromise = new Promise((resolve, reject) => {
+    RCTAEPEdge.sendEvent(experienceEvent)
+    .then(eventHandles => {
+      let eventHandlesPromise = toEventHandle(eventHandles);
+      resolve(eventHandlesPromise);
+    })
+    .catch((error) => {
+      reject(error);
+    });      
+    });
+     return(sentEventPromise);
+  },
 };
+
+
+function toEventHandle(eventArray: Array) {
+ 
+    let edgeEventArray = [];
+
+    if (!eventArray) {
+      return [];
+    }
+
+    for (let prop of eventArray) {
+      edgeEventArray.push(new EdgeEventHandle(prop.type, prop.payload));
+    }
```

Comment:
For L61 -> 67 can we have something like:
if (eventArray) {
for (let prop of eventArray) {
      edgeEventArray.push(new EdgeEventHandle(prop.type, prop.payload));
    }
}